_posts/2020-07-08-Book-Five-Dysfunctions.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
```

Comment:
General remark:

It might be worth mentioning that there is a workbook also that helps you get hands-on with your team